website/docs/ephemeral-resources/azure_secrets_static_credentials.html.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: "vault"
+page_title: "Vault: ephemeral vault_azure_static_credentials data resource"
+sidebar_current: "docs-vault-ephemeral-azure-static-credentials"
+description: |-
+  Read an ephemeral static secret from the Vault Azure Secrets engine
+
+---
+
+# vault_azure_access_static_credentials (Ephemeral)
+
+Reads ephemeral static Azure credentials for a static role managed by the Azure Secrets Engine.  
+These credentials are not stored in Terraform state.
+
+For more information, refer to the [Vault Azure Secrets Engine documentation](https://developer.hashicorp.com/vault/docs/secrets/azure).
+
+## Example Usage
+
+```hcl
+resource "vault_azure_secret_backend" "azure" {
+  subscription_id = var.subscription_id
+  tenant_id       = var.tenant_id
+  client_secret   = var.client_secret
+  client_id       = var.client_id
+}
+
+resource "vault_azure_secret_backend_static_role" "example" {
+  backend              = vault_azure_secret_backend.azure.path
+  role                 = "example-role"
+  application_object_id = "00000000-0000-0000-0000-000000000000"
+}
+
+ephemeral "vault_azure_static_credentials" "creds" {
+  backend = vault_azure_secret_backend.azure.path
+  role    = vault_azure_secret_backend_static_role.example.role
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `backend` - (Required) Path to the mounted Azure Secrets Engine where the static role resides.
```

Comment:
we should add the parameters that are available to all Ephemeral resources by default: `namespace` and `mount_id`. Example docs for `mount_id` [here](https://github.com/hashicorp/terraform-provider-vault/blob/main/website/docs/ephemeral-resources/kv_secret_v2.html.md?plain=1#L62)

Reply:
Good catch! Thank you.

Added here: https://github.com/hashicorp/terraform-provider-vault/pull/2635/commits/c51c4daf72e49b5d7fb3dc908c215ce2715d0fcc

copied `mount_id` from the example you sent above, and copied the `namespace` docs from [azure access credential resource](https://github.com/hashicorp/terraform-provider-vault/blob/main/website/docs/d/azure_access_credentials.html.md?plain=1#L67-L70)